specs/ingestion/common/schemas/event.yml

```
@@ -36,8 +36,8 @@ Event:
 
 EventStatus:
   type: string
-  enum: ['created', 'started', 'retried', 'failed', 'succeeded', 'critical']
+  enum: [created, started, retried, failed, succeeded, critical]
 
 EventType:
   type: string
-  enum: ['fetch', 'record', 'log', 'transform']
+  enum: [fetch, record, log, internal, transform]
```

```suggestion
  enum: [fetch, record, log, transform]
```

and Pierre was right here since this type is only used for filtering it doesn't need to be exposed on the public API